playground/mocks/data/idp/idx/authenticator-enroll-select-authenticator.json

```diff
@@ -209,6 +224,16 @@
             "type": "totp"
           }
         ]
+      }, {
+        "displayName": "Google Authenticator",
+        "type": "app2",
```

Comment:
update to app and then try to see if tests pass.

Reply:
We cannot change it back to app2. it will cause trouble that same icon and description as OV.